tools/helper.py

```diff
@@ -11,3 +12,10 @@ def yaml_read(path):
         except yaml.YAMLError as exc:
             print(exc)
     return yml
+
+
+def remove_html_tags(text):
+    import re
+    tag_re = re.compile(r'<[^\n>-]+>')
+    text_mod = tag_re.sub('', text)
+    return re.sub("\n", "", text_mod)
```

Comment:
Just ran it through a couple examples and it is not removing the `<a href` tags as well as `rel="nofollow"`

Maybe some BeautifulSoup usage would help here? We just need to make sure, to keep the links in a format. Either they stay inline, or we could use something like `[1]` with `[1] $link` at the end

```
before:
'In case of missing credential, please follow these configuration steps \n<a href="https://docs.vmware.com/en/Management-Packs-for-vRealize-Operations-Manager/8.2/sddc-health/GUID-65C5FF79-74E6-41F8-92B5-4E2027B6B8BC.html" rel="nofollow">here</a>.'
                                                                                                                                                                                                                                                                                              
after:
'In case of missing credential, please follow these configuration steps <a href="https://docs.vmware.com/en/Management-Packs-for-vRealize-Operations-Manager/8.2/sddc-health/GUID-65C5FF79-74E6-41F8-92B5-4E2027B6B8BC.html" rel="nofollow">here.'

Reply:
```
Fix the vSphere Security Configuration Guide Rules Violations according 
to the recommendations available at 
<a href="http://www.vmware.com/security/hardening-guides.html" 
rel="nofollow">vSphere Security Configuration Guide</a>.
 ------
Fix the vSphere Security Configuration Guide Rules Violations according 
to the recommendations available at vSphere Security Configuration Guide.
http://www.vmware.com/security/hardening-guides.html
```

Reply:
That looks better.